### PR TITLE
Feature/nuc 697 multiselect

### PR DESCRIPTION
**Pull request by volkanakbayir**

### CODE REVIEW

File: src/components/VxpMultiSelectDropdown/NativeSelectionModal.vue [hunk 1]
```diff
@@ -0,0 +1,90 @@
+<template>
+  <Frame>
```
Comment (ahmetceylan) [inline src/components/VxpMultiSelectDropdown/NativeSelectionModal.vue]: we can move base or helper components into core folder

File: docs/.vuepress/enhanceApp.js [hunk 2]
```diff
@@ -1,11 +1,30 @@
...
     addVxpPlugins(Vue);
+    Vue.component('StackLayout',StackLayout);
+    Vue.component('WrapLayout',WrapLayout);
+    Vue.component('GridLayout',GridLayout);
+    Vue.component('DockLayout',DockLayout);
+    Vue.component('TextField',TextField);
+    Vue.component('ModalDialog',ModalDialog);
+    Vue.component('Label',Label);
+    Vue.component('Button',Button);
+    Vue.component('SearchBar',SearchBar);
```
Comment (ercang) [inline docs/.vuepress/enhanceApp.js]: We shouldn't register these components here.

Comment (volkanakbayir) [reply docs/.vuepress/enhanceApp.js]: if we do not register components there and inside component itself. native build tries to import web components and crashes

Comment (volkanakbayir) [reply docs/.vuepress/enhanceApp.js]: i have moved things according to new design

File: src/components/VxpMultiSelectDropdown/NativeSelectionModal.vue (hunk 1 above)
Comment (volkanakbayir) [reply src/components/VxpMultiSelectDropdown/NativeSelectionModal.vue]: moved

File: docs/.vuepress/components/VxpMultiSelectDropdownLiveDemoDoc.vue [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+      </StackLayout>
+
+      <StackLayout class="margin-top-10" v-for="(item,index) in selectableItems" :key="index" orientation="horizontal">
+        <StackLayout class="m-r-10" orientation="horizontal">
+          <Label class="m-r-10" text="Label: "></Label>
+          <TextField :id="'vxpmultiselectdropdown_items__label_' + index + '_input'" v-model="item.label"></TextField>
+        </StackLayout>
+        <StackLayout class="m-r-10" orientation="horizontal">
+          <Label class="m-r-10" text="Selected: "></Label>
+          <input :id="'vxpmultiselectdropdown_items__selected_' + index + '_checkbox'" type="checkbox" @change="changeStatus($event.target.checked,item)" :checked="isItemSelected(item)">
```
[the remark this answers is not in the document]
Comment (volkanakbayir) [reply docs/.vuepress/components/VxpMultiSelectDropdownLiveDemoDoc.vue]: done

**Pull request merged**